File: WEEK1/seminar/src/main/java/com/server/dosopt/seminar/service/Person.java
package com.server.dosopt.seminar.service;


import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@Builder   // 빌더패턴 적용 (롬복 어노테이션으로 제공)
public class Person {

    private String lastName;
    private String firstName;

    public Person(String lastName, String firstName) {
        this.lastName = lastName;
        this.firstName = firstName;
    }
}

File: WEEK1/kotlin-seminar/src/main/kotlin/sopt/org/kotlinseminar/domain/Person.kt
package sopt.org.kotlinseminar.domain

import lombok.AccessLevel
import lombok.Builder
import lombok.NoArgsConstructor

@NoArgsConstructor(access = AccessLevel.PROTECTED)
@Builder
class Person(
    var lastName: String?,
    var firstName: String?
)   // 자바의 멤버 대신 프로퍼티 형식으로 작성하면 Getter, Setter의 역할까지 한번에 처리해준다